boilerplate/flyte/end2end/run-tests.py

```diff
@@ -188,12 +119,12 @@ def schedule_workflow_groups(
     return results
 
 
-def valid(workflow_group):
+def valid(workflow_group, parsed_manifest):
     """
-    Return True if a workflow group is contained in FLYTESNACKS_WORKFLOW_GROUPS,
+    Return True if a workflow group is contained in parsed_manifest,
     False otherwise.
     """
-    return workflow_group in FLYTESNACKS_WORKFLOW_GROUPS.keys()
+    return workflow_group in set(wf_group["name"] for wf_group in parsed_manifest)
```

Comment:
does this actually work when script is run with `priorities="lite"`? parsed_manifest like this https://raw.githubusercontent.com/flyteorg/flytesnacks/v0.3.257/flyte_tests_manifest.json doesn't contain group with name `lite`

Reply:
it doesn't, fix in https://github.com/flyteorg/flytesnacks/pull/1429.